Unreal/SystemLocalized/det/UPak.det

```diff
@@ -63,15 +63,15 @@ DeathMessage="Die Granate von %k ließ %o explodieren."
 FemDeathMessage="Die Granate von %k ließ %o explodieren."
 FemKillMessage="Die Granate von %k ließ %o explodieren."
 ; EN: PickupMessage="You got the UMS Grenade Launcher"
-PickupMessage="Du hast den UMS Granatwerfer"
+PickupMessage="UMS-Granatwerfer erhalten"
 ; EN: ItemName="UMS Granatenwerfer"
-ItemName="UMS Granatwerfer"
+ItemName="UMS-Granatwerfer"
 
 [Cloak]
 ; EN: ExpireMessage="disengaged."
 ExpireMessage="deaktiviert."
 ; EN: PickupMessage="You got the Cloaking Device"
-PickupMessage="Du hast die Tarneinrichtung"
+PickupMessage="Tarneinrichtung erhalten"
```

Comment:
Also possible: "Tarnkappengerät". Though in that case Invisibility must remain "Unsichtbarkeit" to avoid confusion.